c-bindings/src/data_serializable_raw.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <vector>
+
+namespace apache {
+namespace geode {
+namespace client {
+namespace internal {
+
+class DataSerializableRaw : public DataSerializablePrimitive , public CacheableKey {
+ public:
+  DataSerializableRaw(const int8_t* data, size_t size);
+  ~DataSerializableRaw() noexcept override = default;
+
+  static std::shared_ptr<DataSerializableRaw> create(
+      const int8_t* data, size_t size);
+
+
+  virtual void toData(DataOutput& dataOutput) const override;
```

Comment:
`virtual` is implied by `override`. `clang-tidy` should be complaining about this.

Reply:
Good catch! I guess you're better than clang-tidy.